freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/FreeIpaUsersStateProvider.java

```diff
@@ -66,9 +66,8 @@ public UsersState getFilteredFreeIpaState(FreeIpaClient freeIpaClient, Set<Strin
             if (IPA_PROTECTED_USERS.contains(userName)) {
                 continue;
             }
-            Optional<com.sequenceiq.freeipa.client.model.User> ipaUserOptional = freeIpaClient.userFind(userName);
-            if (ipaUserOptional.isPresent()) {
-                com.sequenceiq.freeipa.client.model.User ipaUser = ipaUserOptional.get();
+            try {
+                com.sequenceiq.freeipa.client.model.User ipaUser = freeIpaClient.userShow(userName);
```

Comment:
you can use `FreeIpaClientExceptionUtil#ignoreNotFoundExceptionWithValue`

Reply:
done